### DISCUSSION TITLE
Schmid Factor FCC for Aluminum

### DISCUSSION THREAD

**Discussion by Freya36910** [mtex-toolbox/mtex#1857] [Ask Anything]
Hello.
I am looking for SF Aluminum, but it give 24slip systems.
```matlab
%% Schmid Factor
sS = slipSystem.fcc(ebsd.CS)

sSAll = sS.symmetrise;
sSLocal = grains.meanOrientation * sSAll
% compute Schmid factor
Force=vector3d(1,0,0);
r=normalize(Force)
sigma = stressTensor.uniaxial(r)
SF = sSLocal.SchmidFactor(sigma);
```
Bests,
Freya

**Comment by Freya36910** [comment]
I should mention when I use ('antipodal') it will be 12 slip systems but result is different compare to channel5 but for 24slip systems is totally same.
Could you please help me in this regard.
Freya

**Comment by ralfHielscher** [comment]
Hi Freya,
may you have forgotten to formulate the question?
Also have you read https://mtex-toolbox.github.io/SchmidtFactor.html

If you use the option `'antipodal'` you should use `abs(sSLocal.SchmidFactor(sigma))`

All the best,
Ralf.

**Comment by Freya36910** [comment]
Many Thanks Ralf. I am working on that.

Have a nice weekend
Freya
